src/sgame/sg_maprotation.cpp

```diff
@@ -1489,12 +1487,29 @@ void G_InitMapRotations()
 	{
 		Log::Warn( "%s file not found.", fileName );
 	}
+}
+
+/*
+===============
+G_InitMapRotations
+
+Load and initialise the map rotations
+===============
+*/
+void G_InitMapRotations()
+{
+	G_LoadMaprotation( "default/maprotation.cfg" );
+	G_LoadMaprotation( "maprotation.cfg" );
```

Comment:
won't this overwrite default/maprotation.cfg?

Reply:
yes and no.
there can be multiple rotation files loaded.
but if there is a defaultRotation1 in maprotation.cfg it could be overwritten.